src/api/courses.js

```diff
@@ -261,26 +261,38 @@ router.patch(
       locals: { userAuthz, course },
     } = res
     const data = matchedData(req)
+    const name = data.name === undefined ? course.name : data.name
```

Comment:
There's gotta be a better abstraction for this stuff 😂 but this all looks good, thanks for taking care of this!

Reply:
LOL I think I was just being dumb; I refactored it to make it look better